sequencer/src/api.rs

```diff
@@ -239,6 +239,18 @@ impl<N: network::Type, Ver: StaticVersionType + 'static, P: SequencerPersistence
     }
 }
 
+impl<
```

Comment:
Whenever we are running, we have the config in memory. So we could consider putting the config in `SequencerContext` and implementing this also for non-storage nodes, the same way we pull other things out of the context like the state signer and event streamer

Reply:
SystemContextHandle already has that field so I think I can get it from there too